### PR TITLE
Create Gsspcln.jp.xml

### PR DESCRIPTION
**Pull request by cschanaj**
`*.gsspcln.jp` have a wildcard DNS record and a wildcard certificate. thanks.

Related: https://github.com/EFForg/https-everywhere/issues/3069#issuecomment-263163532, https://github.com/EFForg/https-everywhere/pull/6096.

### CODE REVIEW

**Comment by J0WI** [comment]
Does this host have any content?

**Comment by cschanaj** [comment]
This host seems to serve some advertisement (mostly JavaScript) content under `geniee.ne.jp`. I have found the following URL(s) from Google (query: `html:gsspcln.jp`)

- https://1540.gsspcln.jp/sdk/t/1219.js

- https://102355.gsspcln.jp/t/007/864/a1007864.js

- https://js.gsspcln.jp/l/jssdk_debug.js

- https://js.gsspcln.jp/t/061/753/a1061753.js

- https://js.gsspcln.jp/t/047/267/a1047267.js

Do you want me to add these test URLs into the ruleset?

File: src/chrome/content/rules/Gsspcln.jp.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!--
+	Remark: 
+	 - *.gsspcln.jp have a wildcard DNS and a wildcard certificate
+
+	 - Since only JavaScript test URLs are found, platform="mixedcontent"
```
Comment (J0WI) [inline src/chrome/content/rules/Gsspcln.jp.xml]: I don't see the context?

Comment (cschanaj) [inline src/chrome/content/rules/Gsspcln.jp.xml]: You're right, I have mistaken the concept of MCB this time. Fixed. Thanks!

**Pull request merged**